### PR TITLE
Alberto/aws db custom sg

### PR DESCRIPTION
**Pull request by The-smooth-operator**

### CODE REVIEW

**Comment by The-smooth-operator** [comment]
Travis failure, not my fault

File: aws/database/main.tf [hunk 1]
```diff
@@ -75,8 +75,8 @@ resource "aws_db_instance" "default" {
   identifier                            = var.identifier
   username                              = local.username
   password                              = random_password.password.result
-  db_subnet_group_name                  = aws_db_subnet_group.subnet.id
-  vpc_security_group_ids                = [aws_security_group.default.id]
+	db_subnet_group_name                  = var.custom_subnet == "" ? aws_db_subnet_group.subnet.id : var.custom_subnet
```
Comment (bowlofstew) [inline aws/database/main.tf:78]: Formatting?

File: aws/database/main.tf [hunk 2]
```diff
@@ -104,6 +104,8 @@ resource "aws_db_instance" "read_replica" {
   parameter_group_name                  = var.parameter_group_name != "" ? var.parameter_group_name : "default.${var.type}${local.ver}"
   apply_immediately                     = true
   skip_final_snapshot                   = "true"
+  vpc_security_group_ids                = [var.custom_replica_sg == "" ? aws_security_group.default.id : var.custom_replica_sg]
+	db_subnet_group_name                  = var.custom_replica_subnet == "" ? aws_db_subnet_group.subnet.id : var.custom_replica_subnet
```
Comment (bowlofstew) [inline aws/database/main.tf:108]: Formatting?

File: aws/database/main.tf (hunk 1 above)
Comment (The-smooth-operator) [reply aws/database/main.tf:78]: thanks for the catch!

File: aws/database/main.tf (hunk 2 above)
Comment (The-smooth-operator) [reply aws/database/main.tf]: thanks for the catch!

**Pull request merged**